README.md

```diff
@@ -222,6 +222,8 @@ Lager is also supplied with a `error_logger` handler module that translates
 traditional erlang error messages into a friendlier format and sends them into
 lager itself to be treated like a regular lager log call. To disable this, set
 the lager application variable `error_logger_redirect` to `false`.
+You can also disable reformating for OTP and Cowboy messages by setting variable
+`error_logger_format_raw` to `true`.
```

Comment:
Typo: "reformatting".

Other than that, this looks fine. I'm not thrilled that `error_logger_lager_h:log_event` just got hairier, but it's not significantly worse than it already was. Problem for another day.

Will give it final approval with the typo fix.